### DISCUSSION TITLE
Disable guest feature

### DISCUSSION THREAD

**Discussion by lucasrochaa** [vendurehq/vendure#1597] [Q&A]
Hello, there is a easy way to disable the guest feature, only allow authenticated in vendure?

thanx!

**Comment by michaelbromley** [comment]
Hi!
You could use a [custom order process](https://www.vendure.io/docs/developer-guide/customizing-the-order-process/) which checks in the `onTransitionStart` hook to see if the user is logged in, and if not it would prevent the state transition.

**Reply by lucasrochaa** [reply]
Thanx Michael!
